.github/workflows/tf_apply_production.yml

```diff
@@ -0,0 +1,44 @@
+name: "Terraform Apply Production"
+
+on:
+  push:
+    branches:
+      - main
+    paths:
+      - "terragrunt/**"
+      - ".github/workflows/tf_apply_production.yml"
+
+env:
+  AWS_REGION: ca-central-1
+  TERRAFORM_VERSION: 1.3.9
+  TERRAGRUNT_VERSION: 0.38.4
+  CONFTEST_VERSION: 0.27.0
```

Comment:
A really minor comment - could you update these tool versions all to their latest releases.